Exercise_2.py

```diff
@@ -5,10 +5,30 @@ def __init__(self, data):
  
 class Stack:
```

Comment:
Implement this SLL Stack, by adding and removing to the head instead of tail. O(1) time for all operations.